test/integration/docker_runtime/test_api_test_environment_default_docker_runtime.py

```diff
@@ -0,0 +1,22 @@
+from test.integration.docker_runtime.common import (
+    assert_container_runtime,
+    default_docker_runtime,
+)
+
+
+def test_test_container_runtime(
+    api_default_database_with_test_conainer_module, default_docker_runtime
```

Comment:
Here is a typo: `conainer` without "T", but should be `container` with a "T".
The fixture is already spelled wrong. 
Also, I don't like the long name, being rather a sentence than a name. 

Could we maybe fix the naming? 
* What is a "test container"? 
* What is a _default_ database? Are there other (non-default) databases as well? 
* What is a _module_? Are there more modules besides the "test container"?

And looking inside the fixture `api_default_database_with_test_conainer_module`:
* it uses the name `provider` (which could rather be `context`) 
* but renames it to `db` 
* The type hint says `ExaslctDockerTestEnvironment` 

I am sorry, but this is a mess.

Reply:
Fixed in [another PR.](https://github.com/exasol/integration-test-docker-environment/pull/480)